assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -27,25 +27,33 @@ import isPlainObject from 'lodash/isPlainObject';
  */
 import API from 'googlesitekit-api';
 import Data from 'googlesitekit-data';
+import { deleteItem, getItem, setItem } from '../../../googlesitekit/api/cache';
 import { STORE_NAME } from './constants';
 import { createFetchStore } from '../../data/create-fetch-store';
 import { actions as errorStoreActions } from '../../data/create-error-store';
-const { commonActions, createRegistrySelector } = Data;
+const { commonActions, createRegistryControl, createRegistrySelector } = Data;
 const { receiveError, clearError } = errorStoreActions;
 
+const CACHE_KEY_NAME = 'fetchGetUserInputSettingsStore.fetchGetUserInputSettingsStore';
+
 function fetchStoreReducerCallback( state, inputSettings ) {
 	return { ...state, inputSettings };
 }
 
 const fetchGetUserInputSettingsStore = createFetchStore( {
 	baseName: 'getUserInputSettings',
-	controlCallback: () => API.get( 'core', 'user', 'user-input-settings', undefined, { useCache: false } ),
+	controlCallback: async () => {
+		return API.get( 'core', 'user', 'user-input-settings', undefined, { useCache: false } );
+	},
 	reducerCallback: fetchStoreReducerCallback,
 } );
 
 const fetchSaveUserInputSettingsStore = createFetchStore( {
 	baseName: 'saveUserInputSettings',
-	controlCallback: ( settings ) => API.set( 'core', 'user', 'user-input-settings', { settings } ),
+	controlCallback: ( settings ) => {
+		return API.set( 'core', 'user', 'user-input-settings', { settings } )
+		;
+	},
```

Comment:
These changes seem unrelated/unnecessary?

Reply:
Ah, I think I was doing something in there with console logs previously, removed them, and that's why the change is there. I've reverted it.